### PR TITLE
Add some colours to light theme, add new token transformer for light

### PR DESCRIPTION
**Pull request by CCAyl**
This MR does a couple things, and i'd like some discussion* on some of them:

*adds `transparent` as a `shade` in our `core colours`
- do we hate this? wasn't sure where else to put it

adds some colours to our `light set` in figma tokens, aka `light.json`
- does this structure make sense in both code AND figma tokens?
- are the colours right?

*updates the build-tokens.yml
- when `light.json` is edited, this will run
- should we consider breaking out github actions per theme? this might end up being a pretty chunky action that runs when it may not need to. will need to consider what actually needs to be ran when.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,9 @@
   ],
   "scripts": {
     "build": "node build.js",
-    "transform-base": "npx token-transformer data data/transformed-core.json core --expandTypography",
-    "transform": "yarn transform-base"
+    "transform-core": "npx token-transformer data data/transformed-core.json core --expandTypography",
+    "transform-light": "npx token-transformer data data/transformed-light.json core,light core",
+    "transform": "yarn transform-core && yarn transform-light"
```
Comment (CCAyl) [inline package.json:13]: do we hate this combo pack script?

File: data/light.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        "value": "{colors.grey.700}",
+        "type": "color"
+      },
+      "subdued": {
+        "value": "{colors.grey.500}",
+        "type": "color"
+      }
+    },
+    "surface": {
+      "surfaceLevel0": {
```
Comment (alexandra-lim) [inline data/light.json:18]: Are we naming this because we are hoping to strip out the `surface` with a transform?

File: package.json (hunk 1 above)
Comment (alexandra-lim) [reply package.json:13]: Nah, but I like `transform-core`! Nice!

File: package.json [hunk 3]
```diff
@@ -8,8 +8,9 @@
   ],
   "scripts": {
     "build": "node build.js",
-    "transform-base": "npx token-transformer data data/transformed-core.json core --expandTypography",
-    "transform": "yarn transform-base"
+    "transform-core": "npx token-transformer data data/transformed-core.json core --expandTypography",
+    "transform-light": "npx token-transformer data data/transformed-light.json core,light core",
```
Comment (alexandra-lim) [inline package.json:12]: I brought this up in our group chat, but we have some instances of `npm` in our GitHub workflow, do we want to stick with `yarn` for everything and remove `package-lock.json`? We can update this in another MR

**Review by alexandra-lim — COMMENTED** [review]
I tested this in the plugin and I'm seeing this in the `light` file. I'm wondering though if it's because I don't have the licence key?

Screen Shot 2022-08-04 at 5 39 35 PM

File: package.json (hunk 3 above)
Comment (CCAyl) [reply package.json:12]: i think that makes sense! seems like a good clean up

**Comment by CCAyl** [comment]
re: figma issues, if you select the core check box, does that fix the issue? if not, you can potato the license key to see if that solves it.


additionally, was thinking we should move the `transform-core` and `transform-light` files to a new folder so they don't appear in figma

File: package.json (hunk 1 above)
Comment (iryanclarke) [reply package.json:13]: Why do we have `core,light`, and then `core`? Is that not just excluding the `core` that we already included?

Comment (iryanclarke) [reply package.json:13]: AHH, okay lol, for some reason I was like "Why do you need to include core", but then I realized its cause `light` has references to core, and thus you need both to transform them

**Review by alexandra-lim — APPROVED** [review]
Checking `core` worked. We can move the transformed tokens into a different folder in another MR, I also like that idea.

LGTM!

File: package.json (hunk 1 above)
Comment (CCAyl) [reply package.json:13]: exactly!

**Pull request merged**